### PR TITLE
Move docs for Candela and Vega to the plugin docs file

### PR DESCRIPTION
**Pull request by brianhelba**
I've aggregated these changes from some other WIP branches of mine, so we can get them live ASAP.

* Update installation documentation references to "npm install"
  * Also, reference a non-existing dependency in plugin documentation.
* Move documentation for the Candela plugin to the plugin docs
* Move documentation for the Vega plugin to the plugin docs

### CODE REVIEW

**Comment by brianhelba** [comment]
@girder/developers PTAL.

File: grunt_tasks/fontello.js [hunk 1]
```diff
@@ -45,13 +46,14 @@ module.exports = function (grunt) {
     } else {
         // Download font archive from data.kitware.com
         grunt.config.merge({
-            shell: {
+            curl: {
                 fontello: {
-                    command: 'curl "' + srcUrl + '" -o "' + archivePath + '"'
```
Comment (jbeezley) [inline grunt_tasks/fontello.js]: We actually used `grunt-curl` before, but we changed it due to flaky downloads.  See the commit here: https://github.com/girder/girder/commit/051cf2b881f9f628546f53a93983841ee9d6f534.

**Comment by brianhelba** [comment]
I've removed the `grunt-curl` changes. I think any potential bugs need to be better documented in the source code and/or issues, but I'll handle that separately.

@girder/developers PTAL

**Comment by zachmullen** [comment]
Could we hold off on this until after the npm packaging changes go in? Or back out the grunt-shell update?

**Comment by brianhelba** [comment]
It can wait.

**Comment by brianhelba** [comment]
@girder/developers @zachmullen I've gone ahead and removed the change to `grunt-shell`, which I'll put in another PR.  PTAL now.

**Pull request merged**